### ISSUE TITLE
Why is Internationalization support disabled?

### ISSUE THREAD

**Issue by johnbradley** [rocker-org/rocker-versioned2#203]
The following line disables `Natural language support` when configuring R:
https://github.com/rocker-org/rocker-versioned2/blob/b6b64306237ac0d31742b3f1ae31820a9b4a02f6/scripts/install_R.sh#L128

See [R Internationalization configuration docs](https://cran.r-project.org/doc/manuals/r-release/R-admin.html#Internationalization-support). 

I was wondering the reason behind disabling this feature.

Thanks.

**Comment by cboettig** [comment]
Thanks @johnbradley .  I think we chose most of these flags based on matching how Debian builds the `r-base` binary upstream, but maybe I'm just inventing that.  @eddelbuettel probably has deeper insights on that one.  I'd be happy to drop that flag unless there are any objections!

**Comment by eddelbuettel** [comment]
Hm, no idea.  _Could be_ a case of a simple cookie-cutter copying of other setups where I had scripts for compiling R(-deve) from source.  Could have been motivated by making the image smaller -- we often have people coming complaining that we do not used "shrunk / minimized" Debian/Ubuntu or even Alpine.

On the margin, I'd just spin up a local variant of the Dockerfile, enable nls and enjoy.  We've been doing since 2014 or so and it is to my knowledge the first request for i18n so our default build seems to work for a number of people.

**Comment by terashim** [comment]
Hi.

[pkgdown 2.0 now supports translations](https://www.tidyverse.org/blog/2021/12/pkgdown-2-0-0/) which depends on nls. It does not work on the image which disabled nls.
It would be useful if nls is enabled to use the translation feature.

Thank you!

**Comment by eddelbuettel** [comment]
What would `pkgdown` have to do with how we build Docker containers?

**Comment by cboettig** [comment]
@terashim thanks for the ping!  I've updated the rocker-versioned script to no longer disable nls. (@eddelbuettel 's R-devel script doesn't disable nls, https://github.com/rocker-org/r-devel/blob/master/Dockerfile, so not sure how I managed to introduce the disable in the first place!)

Fix should be picked up by the autobuilds. 

 @eitsupi do we need to do anything to break caching for the autobuild to pick up changes in `scripts/` now?

**Comment by eitsupi** [comment]
You do not need to do anything other than update the file.

Since the hash of `COPY scripts/install_R.sh /rocker_scripts/install_R.sh` is a different value, the cache is not used in the layer where R is installed (`RUN /rocker_scripts/install_R.sh`), and the build proceeds.

https://github.com/rocker-org/rocker-versioned2/blob/8a92ac09f69fbc414d69e571d68ceb54953620f4/dockerfiles/r-ver_4.1.2.Dockerfile#L14-L16

**Comment by eitsupi** [comment]
There seemed to be very little increase in image size (851MB -> 852MB).

https://github.com/rocker-org/rocker-versioned2/wiki/r-ver_8a031e25eda6
https://github.com/rocker-org/rocker-versioned2/wiki/r-ver_8a031e25eda6

**Comment by terashim** [comment]
@cboettig Thank you for updating! I have confirmed that the pkgdown translation now works on the updated image :)

**Comment by lcougnaud** [comment]
Would that be possible to also update the other r-ver images with R > 4.0.0?
I would like to build a Docker image based on r-ver 4.0.5, and the disabling of NLS generates quite some warnings in tests generated with `testthat`:
```
Warning (test_....R): Changing language has no effect when R installed without NLS
Backtrace:
 1. testthat::expect_true(...)
 2. testthat:::expect_waldo_constant(act, TRUE, info = info)
 3. testthat:::waldo_compare(...)
 4. reporter$local_user_output()
 5. testthat::local_reproducible_output(...)
 6. withr::local_language(lang, .local_envir = .env)
 ```
 Thanks in advance!

**Comment by eitsupi** [comment]
Closed by 8a92ac09f69fbc414d69e571d68ceb54953620f4 and all images will be updated today.

@lcougnaud Please pull image from DockerHub and check it.

```shell
$ docker run --rm -it --pull always rocker/r-ver:4.0.0
4.0.0: Pulling from rocker/r-ver
7c3b88808835: Pull complete
df3a04b8ec6d: Pull complete
8529b06fe874: Pull complete
4d67221a2029: Pull complete
960ba66be918: Pull complete
Digest: sha256:b4ce816d20209ffa3892bc0f9c2c7fe79a037fee8b897a5787d956928be50fc9
Status: Downloaded newer image for rocker/r-ver:4.0.0

R version 4.0.0 (2020-04-24) -- "Arbor Day"
Copyright (C) 2020 The R Foundation for Statistical Computing
Platform: x86_64-pc-linux-gnu (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.
```

**Comment by lcougnaud** [comment]
Thanks for the update!
The issue is indeed solved when using the new Docker images (from 10/03/2022).
